tests/plugins/widget/editing.js

```diff
@@ -335,6 +360,24 @@
 			} );
 		},
 
+		// (#1044)
+		'test opening and hiding dialog events': function() {
```

Comment:
This test is not checking events themselves, but the fact if widget is passed to them. Therefore the name is misleading.